homeassistant/components/mullvad/__init__.py

```diff
@@ -0,0 +1,68 @@
+"""The Mullvad VPN integration."""
+import asyncio
+from datetime import timedelta
+import logging
+
+import async_timeout
+from mullvad_api import MullvadAPI
+
+from homeassistant.config_entries import ConfigEntry
+from homeassistant.core import HomeAssistant
+from homeassistant.helpers import update_coordinator
+
+from .const import DOMAIN
+
+PLATFORMS = ["binary_sensor", "sensor"]
```

Comment:
Please limit initial PRs for new integrations to a single platform.

Reply:
🤐 Maybe i shouldn't ask, but is there a reason for requiring this? It seems like it only stifles contributions.

Reply:
It makes PRs smaller, easier to review, faster to push thru. Follow up PRs will be smaller as well.

Reply:
Thanks for the explanation. Sounds reasonable to me. Now i need to decide which sensors are more important 🤔. The one binary sensor is probably the most important, but it's only 1 vs 9 sensors! We'll go with BinarySensors first.